spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -653,7 +657,7 @@ function recvPacket(
 
     channel = provableStore.get(channelPath(packet.destPort, packet.destChannel))
     abortTransactionUnless(channel !== null)
-    abortTransactionUnless(channel.state === OPEN)
+    abortTransactionUnless(channel.state === OPEN || channel.flushStatus === FLUSHING)
```

Comment:
If there is a counterparty `lastPacketSent` stored, should we also abort transaction in `recvPacket` unless `packet.sequence` is <= `lastPacketSent`? At the moment I think `lastPacketSent` is stored in `startFlushUpgradeHandshake` and deleted when the upgrade completes/cancels/timeouts, but it is not used anywhere, right?

Reply:
Is this check required at all? We want this check to exist on `recvPacket` to process packets while flushing, but we don't want to be able to send more packets when we are flushing.